app/js/lib/foxbox.js

```diff
@@ -10,6 +10,11 @@ import FoxboxDb from './foxbox-db';
 const settings = new FoxboxSettings();
 const db = new FoxboxDb();
 
+const privateMembers = {
```

Comment:
Mmm, I've already asnwered why ```private``` is not possible :) [1]

[1] https://github.com/fxbox/app/pull/16#discussion_r56576304